### PR TITLE
Make some homepage copy changes

### PR DESCRIPTION
**Pull request by MatthewStanciu**
Change the copy for "Curriculum", replace "Workshop Bounty" with Meetings, change "resources this semester".

I'm not feeling 100% on the copy changes I made. I think they can all be better. @maxwofford & @sampoder, I would love both of your thoughts if you have some time.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -384,7 +384,7 @@ const Page = () => (
             icon="sam"
             color="blue"
             name="Free Zoom Pro"
-            desc="A paid Zoom subscription for your club to run online meetings without the time limit."
+            desc="24/7 access to Zoom Pro enabled meeting rooms for your club (that means no time limit)."
```
Comment (sampoder) [inline pages/index.js:387]: Thoughts on this?

Or how about:

```suggestion
            desc="24/7 access to Zoom Pro enabled meeting rooms for your club (finally, no time limit)."
```

**Comment by lachlanjc** [comment]
#### 📝Changed routes:


| [**/**](https://v3-et6pch1f6.hackclub.dev/) |
|:-:|
|<a href="https://v3-et6pch1f6.hackclub.dev/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/nhEfBTm3PELqh_-3fSKt0MfueKd-k1wuAohKAtVMnEo2u2AkzQjoE8x5SC9fTGwY1P-GLMEwNp45mm8XD5noGg.png">(view full size)</a> |







Commit e3e532cfb4aca7286e0c688b399776983711e833 (https://v3-et6pch1f6.hackclub.dev).

File: pages/index.js [hunk 2]
```diff
@@ -453,24 +452,21 @@ const Page = () => (
             />
           </Feature>
           <Feature
-            name="Workshop Bounty Program"
+            name="A comprehensive leader guide"
```
Comment (sampoder) [inline pages/index.js:455]: I don't know about the word guide... that might connote something different for others

File: pages/index.js (hunk 1 above)
Comment (MatthewStanciu) [reply pages/index.js:387]: I kinda prefer the first one ("that means no time limit")

File: pages/index.js [hunk 3]
```diff
@@ -453,24 +452,21 @@ const Page = () => (
...
               <>
-                <Highlight>
-                  Get paid $200{' '}
-                  <a href="https://workshops.hackclub.com/workshop-bounty">
-                    for submitting curriculum
-                  </a>
-                </Highlight>{' '}
-                you make for your club for every club to use. Fund your club, or
-                have fun with your bounties.
+                Need help getting started? Watch real club leaders run meetings, and
```
Comment (sampoder) [inline pages/index.js:458]: _omg REAL club leaders_, this one is works really well imo

File: pages/index.js [hunk 4]
```diff
@@ -437,10 +437,9 @@ const Page = () => (
             name="Curriculum"
             desc={
               <>
-                We’re dropping a new{' '}
-                <a href="https://workshops.hackclub.com/">workshop</a> every
-                Monday afternoon, and you can join our Protoclub meeting to see
-                how to run it live.
+                Come prepared to every meeting with over 100{' '}
```
Comment (sampoder) [inline pages/index.js:440]: Something doesn't feel right about this one tbh: idk but it sounds like you are coming to every meeting with over 100 workshops in my head lol?

Comment (sampoder) [reply pages/index.js:440]: How about splitting it into two sentences?

```
We've got over 100 workshops that you can deliver with ease.  Each of them guides your club members through making a unique project.
```

File: pages/index.js [hunk 5]
```diff
@@ -354,7 +354,7 @@ const Page = () => (
       >
         <Box as="header" sx={{ textAlign: [null, 'center'], pb: [4, 5] }}>
           <Text as="p" variant="eyebrow">
-            Resources this semester
+            Crazy useful resources
```
Comment (sampoder) [inline pages/index.js:357]:
```suggestion
            some Crazy useful resources
```

File: pages/index.js (hunk 2 above)
Comment (sampoder) [reply pages/index.js:455]: hmm

Comment (sampoder) [reply pages/index.js:455]: Take inspiration from the pros? lol no that's too wordy

Comment (sampoder) [reply pages/index.js:455]: Ace the First Meeting?

Comment (sampoder) [reply pages/index.js:455]: Learn from the pros? hmm but I dont want it to sound exclusive

Comment (sampoder) [reply pages/index.js:455]: Learn from fellow leaders

Comment (sampoder) [reply pages/index.js:455]: Get Inspired

**Comment by MatthewStanciu** [comment]
I'm going to merge this because I haven't been able to come up with anything that really excited me, and I think these are pretty necessary changes to make to the website, but I'm still very open to feedback ([started a thread in #code](https://hackclub.slack.com/archives/C0EA9S0A0/p1617982108017500))
